README.md

```diff
@@ -7,6 +7,9 @@ Graphql-java add-on for orchestrating GraphQL services.
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=dotwebstack_graphql-java-orchestrate&metric=alert_status)](https://sonarcloud.io/dashboard?id=dotwebstack_graphql-java-orchestrate)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=dotwebstack_graphql-java-orchestrate&metric=coverage)](https://sonarcloud.io/dashboard?id=dotwebstack_graphql-java-orchestrate)
 
+# Deprecation notice:
+Since the beginning of this repo it has mainly been maintained by people working for ‘Het Kadaster’. Unfortunately it hasn’t been widely used as we hoped to. That has lead to the decision to stop maintaining this repo in its current public form. From this day forward, 14th January 2025, there will be no active support for this repo and the repo will be archived.
```

Comment:
Voorstel:
Since the inception of this repository, it has primarily been maintained by people working at ‘Het Kadaster’. Unfortunately, it has not been used as widely as we had hoped. As a result, we have decided to stop maintaining this repository in its current public form. Starting from today, January 14, 2025, there will be no active support for this repository, and it will be archived.